tests/tdk_lambda_genesys.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from unittest import skipIf
+
+from utils.channel_access import ChannelAccess
+from utils.testing import get_running_lewis_and_ioc
+from utils.ioc_launcher import IOCRegister
+
+# MACROS to use for the IOC
+# Macros may need to be formally set - check dbLoadRecords line in *ioc*_ddevsim_ioc.log file
+MACROS = { "ADDR1": "1", "PORT1": "1"}
+
+
+class OutputMode(object):
+    VOLTAGE = "VOLTAGE"
+    CURRENT = "CURRENT"
+
+
+class Tdk_lambda_genesysTests(unittest.TestCase):
+
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc("tdk_lambda_genesys")
+        self.ca = ChannelAccess(default_timeout=10)
+        self.ca.wait_for("GENESYS_01:1:VOLT", timeout=30)
+
+    def _write_voltage(self, expected_voltage):
+        self._lewis.backdoor_set_on_device("voltage", expected_voltage)
+        self.ca.set_pv_value("GENESYS_01:1:VOLT", expected_voltage)
+
+    def _write_current(self, expected_current):
+        self._lewis.backdoor_set_on_device("current", expected_current)
+        self.ca.set_pv_value("GENESYS_01:1:CURR", expected_current)
+
+    def _set_powerstate(self, expected_power):
+        self._lewis.backdoor_set_on_device("powerstate", expected_power)
```

Comment:
Personally I'd prefer this be `expected_state`. I implicitly expect `expected_power` to be a power (i.e. in Watts) rather than a binary on/off state.